sycl/CMakeLists.txt

```diff
@@ -162,6 +162,8 @@ add_library("${SYCLLibrary}" SHARED
   "${sourceRootPath}/sampler.cpp"
   "${sourceRootPath}/stream.cpp"
   "${sourceRootPath}/spirv_ops.cpp"
+  "${sourceRootPath}/detail/usm/opencl_shim.cpp"
```

Comment:
Looks like files are listed in alphabetical order, could you please move these two new files at the appropriate place?

Reply:
Yep.